DataConnectors/esetSmc.json

```diff
@@ -0,0 +1,117 @@
+{
+    "id": "EsetSMC",
+    "title": "Eset Security Management Center",
+    "publisher": "Eset",
+    "descriptionMarkdown": "Connector to bring Eset SMC data into Sentnel using custom table for easy quering. Data is sent from Eset SMB in JSON format to syslog connector VM and FluentD parsing is used so data are sent structured to Sentinel.",
```

Comment:
Please ensure it's Azure Sentinel everywhere. Here there's a typo as well. Is it possible to provide details on what type of Eset data is available instead of providing info on the how the data is sent? The how part is covered in the configuration section.